src/dotty/tools/dotc/core/TypeErasure.scala

```diff
@@ -241,10 +232,13 @@ object TypeErasure {
 import TypeErasure._
 
 /**
- *  This is used as the Scala erasure during the erasure phase itself
-   *  It differs from normal erasure in that value classes are erased to ErasedValueTypes which
-   *  are then later converted to the underlying parameter type in phase posterasure.
- *
+ *  @param isJava        Arguments should be treated the way Java does it
```

Comment:
This was just moved from the documentation of `erasureFn`, I don't know enough about java erasure to improve this comment myself :).